src/Commands/RunScan.ts

```diff
@@ -215,7 +214,13 @@ export class RunScan implements CommandModule {
 
       process.exit(0);
     } catch (e) {
-      logger.error(`Error during "scan:run": ${e.error || e.message}`);
+      const errMessage =
+        isAxiosError(e) &&
+        typeof e.response?.data === 'string' &&
+        (e.response?.status === 400 || e.response?.status === 429)
```

Comment:
```suggestion
```
I think we can display errors from any status codes, nope? Here's 403 error message when token has insufficient scopes
<img width="1187" alt="Screenshot 2024-08-14 at 11 19 57 AM" src="https://github.com/user-attachments/assets/58924be0-e9bf-4ed6-b346-52d77142ff30">
at least all 400s seems valid(client error), @bararchy wdyt?
imo if some API errors will be reported as "not human readable" then it's the reason to change API error message and not CLI

Reply:
I checked all error messages from the codes 400 and 429 and verified that in all the cases we throw human readable error from the backend. In case we allow to display the error from every code, sensitive information can be read by the customer

Reply:
if sensitive information returned in API error messages then it's [excessive data exposure](https://owasp.org/API-Security/editions/2019/en/0xa3-excessive-data-exposure/) vulnerability on API side that should be fixed on API side

Reply:
@denis-maiorov-brightsec yeha I like it :) good idea